<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTypes</class>
 <widget class="QWidget" name="FormTypes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Постфиксы</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="DbViewer" name="tableViewBukv"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Варианты</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="DbViewer" name="tableViewVar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Типы по ISO</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="DbViewer" name="tableViewIso"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Типы по AWS</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="DbViewer" name="tableViewAws"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Знаменатели</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="DbViewer" name="tableViewZnam"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Типы по ГОСТ</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="DbViewer" name="tableViewGost"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DbViewer</class>
   <extends>QTableView</extends>
   <header>db/dbviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
